tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -0,0 +1,51 @@
+// <copyright file="LambdaCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Net;
+
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS
+{
+    internal class LambdaCommon
+    {
+        private const string TraceContextEndpointEnvName = "_DD_TRACE_CONTEXT_ENDPOINT";
+        private const string TraceContextEndpoint = "http://127.0.0.1:8124/lambda/trace-context";
+        private const string PlaceholderServiceName = "placeholder-service";
+        private const string PlaceholderOperationName = "placeholder-operation";
+
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(LambdaCommon));
+
+        internal static Scope CreatePlaceholderScope(Tracer tracer)
+        {
+            Scope scope = null;
+            try
+            {
+                string endpointFromEnv = EnvironmentHelpers.GetEnvironmentVariable(TraceContextEndpointEnvName);
+                string endpoint = endpointFromEnv != null ? endpointFromEnv : TraceContextEndpoint;
```

Comment:
[nit] You can remove the line above and do it in one run. 
```suggestion
                var endpoint = EnvironmentHelpers.GetEnvironmentVariable(TraceContextEndpointEnvName) ?? TraceContextEndpoint;
```